### ISSUE TITLE
FUNC_CALL is detected on WINAPI (__stdcall) calls if the type is after WINAPI. If it's after the function is properly marked as FUNC_DEF.

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1255]
```
static LONG WINAPI CustomUnhandledExceptionFilter(EXCEPTION_POINTERS * pExInfo)
{
    if (EXCEPTION_BREAKPOINT == pExInfo->ExceptionRecord->ExceptionCode) // Breakpoint. Don't treat this as a normal crash.
        return EXCEPTION_CONTINUE_SEARCH;
}
```

also need to check this as well
```
		typedef DWORD (WINAPI *SetDllDirectory) (LPCSTR);
```

For now, we've just hacked around it by set COMMENT WINAPI just like for all other __xxxcall.

We want to have a proper solution though or at least raise the issue on github side.

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60006-UNI-2049.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60006-UNI-2049.cpp"
index 4609fbe..600837d 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60006-UNI-2049.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60006-UNI-2049.cpp"
@@ -1,8 +1,8 @@
 // Pointer mark should be formatted (WINAPI* SetXX)
-typedef DWORD (WINAPI* SetDllDirectory)(LPCSTR);
+typedef DWORD (WINAPI * SetDllDirectory)(LPCSTR);
 // Pointer mark should be formatted (EXCEPTION_POINTERS* pExt)
-static LONG WINAPI CustomUnhandledExceptionFilter(EXCEPTION_POINTERS* pExInfo)
+static LONG WINAPI CustomUnhandledExceptionFilter(EXCEPTION_POINTERS * pExInfo)
 {
     if (EXCEPTION_BREAKPOINT == pExInfo->ExceptionRecord->ExceptionCode) // Breakpoint. Don't treat this as a normal crash.
         return EXCEPTION_CONTINUE_SEARCH;
 }
```

Debug support files:
---
[60006-UNI-2049.cpp](https://gist.githubusercontent.com/mihaipopescu/aeee33e02ba163162de00818378ac556/raw/4609fbeaac8268837990663dfafae7b6f7e703c7/60006-UNI-2049.cpp)
[60006-UNI-2049.cpp.log](https://gist.githubusercontent.com/mihaipopescu/aeee33e02ba163162de00818378ac556/raw/f5e5ff4ea305bc11d86d5b29b7065b6a12ff6887/60006-UNI-2049.cpp.log)
[60006-UNI-2049.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/aeee33e02ba163162de00818378ac556/raw/1bfb7f348e3809a12b120fbbc29fcca40c1cda10/60006-UNI-2049.cpp.unc)
[Uncrustify.Common-CStyle.cfg](https://gist.githubusercontent.com/mihaipopescu/aeee33e02ba163162de00818378ac556/raw/33086ee468cf035719d22c1a0feb133c9df40337/Uncrustify.Common-CStyle.cfg)
[UNI-2049.cpp](https://gist.githubusercontent.com/mihaipopescu/aeee33e02ba163162de00818378ac556/raw/7c64f76a1da588be71fd84deb01056876cd6ba12/UNI-2049.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
For the second star use:
```sql
# Add or remove space before pointer star '*'
sp_before_ptr_star              = remove   # ignore/add/remove/force
```

**Comment by gmaurel** [comment]
Fixed 337b538da318f3bec5a4ca5be64e4ad2af784552, Dec 28 16:02:40 2017
